page-components/links.md

```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
   </ul>
 </div>
 
+## Behavior
+
+### Default links
+In general, links should default to opening in the same page or browser tab.
+
+### Opening a link in a new tab
+Links should open in a new tab in interactions where users enter data or make
+selections that would be lost if they left the page and used the back button to
+come back to the page. This includes interactive tools, search filters, and
+forms where the user has to submit and enter information.
```

Comment:
"submit and enter information" is redundant, or at the very least backwards.

Reply:
switched